src/shared/components/Tabs/Tabs.styles.tsx

```diff
@@ -1,16 +1,34 @@
 import styled from '@emotion/styled'
-import { colors } from '@/shared/theme'
-
-export const TabsGroup = styled.div`
-  display: flex;
-`
+import { colors, zIndex } from '@/shared/theme'
 
 type TabProps = {
   selected: boolean
 }
 
+type BackgroundGradientProps = {
+  direction: 'prev' | 'next'
+}
+
+export const TAB_WIDTH = 120
+
+export const TabsWrapper = styled.div`
+  position: relative;
+  width: 100%;
+`
+
+export const TabsGroup = styled.div`
+  display: flex;
+  position: relative;
+  scroll-behavior: smooth;
+  overflow: auto;
+  ::-webkit-scrollbar {
+    display: none;
```

Comment:
scrollbar in Firefox is still visible, so maybe you could something like `scrollbar-width: none;` there